### DISCUSSION TITLE
Is there a way to get the custom-elements.json file without using NPM?

### DISCUSSION THREAD

**Discussion by dannydel** [tyler-technologies-oss/forge#702] [Q&A]
Hello! I am currently working on generation event handling based on the custom-elements.json. So far I can do it if I have the file pulled from NPM but I would rather do it remotely. Is there a way for me to get that?

**Comment by DRiFTy17** [comment]
@dannydel I just came across this, sorry it was missed. I must have not have received a notification for it.

Hopefully you found an answer for it, but currently we do not host the `custom-elements.json` file on the Forge CDN, but you could access it from other public CDNs such as unpkg or jsdelivr:
https://cdn.jsdelivr.net/npm/@tylertech/forge/custom-elements.json
